python-template/{{cookiecutter.placeholder_repo_name}}/tox.ini

```
@@ -0,0 +1,72 @@
+[tox]
+envlist = py{35,36,3738}
+
+[doc8]
+max-line-length = 120
+
+[pycodestyle]
+exclude = .git,.tox
+max-line-length = 120
+
+[pydocstyle]
+; D101 = Missing docstring in public class
+; D200 = One-line docstring should fit on one line with quotes
+; D203 = 1 blank line required before class docstring
+; D212 = Multi-line docstring summary should start at the first line
+; D215 = Section underline is over-indented (numpy style)
+; D404 = First word of the docstring should not be This (numpy style)
+; D405 = Section name should be properly capitalized (numpy style)
+; D406 = Section name should end with a newline (numpy style)
+; D407 = Missing dashed underline after section (numpy style)
+; D408 = Section underline should be in the line following the section’s name (numpy style)
+; D409 = Section underline should match the length of its name (numpy style)
+; D410 = Missing blank line after section (numpy style)
+; D411 = Missing blank line before section (numpy style)
+; D412 = No blank lines allowed between a section header and its content (numpy style)
+; D413 = Missing blank line after last section (numpy style)
+; D414 = Section has no content (numpy style)
+ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
+
+
+[pytest]
+addopts = --cov {{ cookiecutter.sub_dir_name }} --cov-report term-missing --cov-report xml
+norecursedirs = .* docs requirements
+
+[testenv]
+deps =
+    -r{toxinidir}/requirements/test.txt
+commands =
+    pytest {posargs}
+
+[testenv:docs]
+setenv =
+    PYTHONPATH = {toxinidir}
+whitelist_externals =
+    make
+    rm
+deps =
+    -r{toxinidir}/requirements/doc.txt
+commands =
+    doc8 --ignore-path docs/_build README.rst docs
+    rm -f docs/{{ cookiecutter.sub_dir_name }}.rst
+    rm -f docs/modules.rst
+    make -C docs clean
+    make -C docs html
+    python setup.py check --restructuredtext --strict
+
+[testenv:quality]
+whitelist_externals =
+    make
+    rm
+    touch
+deps =
+    -r{toxinidir}/requirements/quality.txt
+commands =
+    touch tests/__init__.py
+    pylint {{ cookiecutter.sub_dir_name }} tests test_utils manage.py setup.py
```

```suggestion
    pylint {{ cookiecutter.sub_dir_name }} tests test_utils *.py
```
Will get `test_settings.py`, along with any other root-level Python scripts.

Same commment for pycodestyle, pydocstyle.